recipes-debian/sbsigntool/sbsigntool/0001-configure-Fixup-build-dependencies-for-cross-compili.patch

```diff
@@ -0,0 +1,54 @@
+From c3533b8da1e1425801d2fc0bcd231e13d593f16b Mon Sep 17 00:00:00 2001
```

Comment:
Where did this patch come from ? I couldn't find exact this patch in [meta-luv](http://git.yoctoproject.org/cgit/cgit.cgi/meta-luv/tree/recipes-devtools/sbsigntool) nor [luv-yocto/meta-luv](https://github.com/intel/luv-yocto/tree/warrior/meta-luv/recipes-devtools/sbsigntool/sbsigntool).

Reply:
https://lists.01.org/hyperkitty/list/luv@lists.01.org/thread/RY2WY2COCNZ6QD2HCZI2XBQO5VE7U6QK/
I applied the patch based on the information above and the patch applied by meta-intel's sbsigntool, and it solved the problem.

Reply:
@L38s I got it. This recipe is also based on the patch [[PATCH] sbsigntool: Use native sysroot to build], right? If so, please describe it so by comment in the recipe.